python/pyhail/dataset.py

```diff
@@ -451,7 +499,14 @@ def annotate_variants_vds(self, other, code=None, root=None):
                 self.jvds, other.jvds, code, root))
 
     def cache(self):
-        """Cache in memory.  cache is the same as persist("MEMORY_ONLY")."""
+        """Mark this dataset to be cached in memory.  ``cache()`` is the same
```

Comment:
Change cache to :py:meth:\`~pyhail.VariantDataset.cache\` and persist to :func:\`persist("MEMORY_ONLY") <pyhail.VariantDataset.persist>\`